.github/workflows/win_exe_sign_test.yml

```
@@ -0,0 +1,47 @@
+
+jobs:
+  build:
+    runs-on: windows-latest
+    env:
+      JKS_B64: ${{ secrets.JARSIGNER_REL_KEYSTORE_B64 }}
+      JKS_PASS: ${{ secrets.JARSIGNER_REL_STOREPASS }}
+      ALIAS: ${{ secrets.JARSIGNER_REL_ALIAS }}
+      PFX_PASS: ${{ secrets.WIN_CERTIFICATE_PWD }}
+    
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)